app/win/main_tl.js

```diff
@@ -1621,11 +1621,17 @@ var loadSavedWindowsFromData = (function(){
     ///     - bordered:<truthy> (default false) to mark windows with borders
     function loadSaveDataV1(data) {
         if(!data.tree) return false;
-        //log.info({'loadSaveDataV1':data});
+        const blankTabsRawUrl = ['about:blank', 'chrome://newtab'];
+        // log.info({'loadSaveDataV1':data});
         let numwins=0;
         for(let win_data_v1 of data.tree) {
-            createNodeForClosedWindowV1(win_data_v1);
-            ++numwins;
+            const notOnlyBlankTabs = win_data_v1.tabs.every(({ raw_url }) => {
+                return ~blankTabsRawUrl.indexOf(raw_url);
```

Comment:
- Unfortunately, this is the wrong sense of the test!  This is testing that every `raw_url` _is_ in `blankTabsRawUrl`.<sup>&dagger;</sup>  When I tried this code it removed all of my saved tabs.  Thankfully, I had a backup :) .
- Also, we do not need to test every item; we only need one non-blank for this window to count.  How about something like this? &mdash;
  ```
  const doOpenWindow = typeof( 
       win_data_v1.tabs.find( ({raw_url}) => { return !blankTabsRawUrl[raw_url]; } ) 
  ) !== 'undefined';
  ```
  That is, as long as we find one tab that is not in `blankTabsRawUrl`, we can proceed to open the window.  What do you think?

<sup>&dagger;</sup> I had to look up the `~` syntax, having never seen it.  I found [this](https://wsvincent.com/javascript-tilde/), which indicates that `~foo.indexOf(...)` is truthy if the element is in the array.